public/app/plugins/panel/canvas/CanvasContextMenu.tsx

```diff
@@ -87,6 +87,14 @@ export const CanvasContextMenu = ({ scene }: Props) => {
           }}
           className={styles.menuItem}
         />
+        <MenuItem
+          label="Edit Canvas"
+          onClick={() => {
+            scene.inlineEditingCallback!();
+            closeContextMenu();
+          }}
+          className={styles.menuItem}
+        />
```

Comment:
It might be nice if the inline edit pop-up was already open that this option could switch to close the pop up with something like `Close Editor`

Reply:
yep!